oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java

```diff
@@ -35,88 +40,127 @@
     public static final String ENDPOINT_NAME = "endpoint_name";
     public static final String ENDPOINT_ID = "endpoint_id";
     public static final String TRACE_ID = "trace_id";
+    public static final String TRACE_SEGMENT_ID = "trace_segment_id";
+    public static final String SPAN_ID = "span_id";
     public static final String IS_ERROR = "is_error";
-    public static final String STATUS_CODE = "status_code";
     public static final String CONTENT_TYPE = "content_type";
     public static final String CONTENT = "content";
+    public static final String DATA_BINARY = "data_binary";
     public static final String TIMESTAMP = "timestamp";
+    public static final String TAGS = "tags";
 
     @Setter
     @Getter
     @Column(columnName = SERVICE_ID)
-    private int serviceId;
+    private String serviceId;
     @Setter
     @Getter
     @Column(columnName = SERVICE_INSTANCE_ID)
-    private int serviceInstanceId;
+    private String serviceInstanceId;
     @Setter
     @Getter
     @Column(columnName = ENDPOINT_ID)
     private String endpointId;
     @Setter
     @Getter
-    @Column(columnName = ENDPOINT_NAME)
+    @Column(columnName = ENDPOINT_NAME, matchQuery = true)
     private String endpointName;
     @Setter
     @Getter
-    @Column(columnName = TRACE_ID)
+    @Column(columnName = TRACE_ID, length = 150)
     private String traceId;
     @Setter
     @Getter
-    @Column(columnName = IS_ERROR)
-    private int isError;
+    @Column(columnName = TRACE_SEGMENT_ID, length = 150)
+    private String traceSegmentId;
     @Setter
     @Getter
-    @Column(columnName = STATUS_CODE)
-    private String statusCode;
+    @Column(columnName = SPAN_ID)
+    private int spanId;
     @Setter
     @Getter
-    @Column(columnName = CONTENT_TYPE)
+    @Column(columnName = IS_ERROR)
+    private int isError;
+    @Setter
+    @Getter
+    @Column(columnName = CONTENT_TYPE, storageOnly = true)
     private int contentType = ContentType.NONE.value();
     @Setter
     @Getter
-    @Column(columnName = CONTENT)
+    @Column(columnName = CONTENT, length = 1_000_000, matchQuery = true)
     private String content;
     @Setter
     @Getter
     @Column(columnName = TIMESTAMP)
     private long timestamp;
 
+    /**
+     * All tag binary data, used when querying.
+     */
+    @Setter
+    @Getter
+    @Column(columnName = DATA_BINARY, storageOnly = true)
+    private byte[] dataBinary;
```

Comment:
I am a little confused about `dataBinary`, `tags`, and `tagsRawData`.
I could understand 2 of them should be raw data and search tags, then what is the 3rd?

Reply:
The `dataBinary` is the raw data for all the tags, `tags` and `tagsRawData` are search tags. `tagsRawData` is used by H2 and MySQL storage, I should rename the `tags` and `tagsRawData` to `searchTags` and `searchTagsRawData`